omod/src/main/resources/config.xml

```diff
@@ -158,7 +158,13 @@
 		<description>
 			concept Id for TRANSFER IN FROM
 		</description>
-	</globalProperty>
+	</globalProperty>	
+		
+	<require_modules>
+	    <require_module version="${mohorderentrybridgeVersion}">
+			org.openmrs.module.mohorderentrybridge

```

Comment:
I did not see you sing this module in your calls into the api though you required it, can you check and see where you were supposed to have used it!